airbyte-ci/connectors/CONNECTOR_CHECKLIST.yaml

```diff
@@ -0,0 +1,12 @@
+paths:
+  "airbyte-integrations/connectors/**":
+    - Connector version has been incremented in the Dockerfile and metadata.yaml according to our [Semantic Versioning for Connectors](https://docs.airbyte.com/contributing-to-airbyte/#semantic-versioning-for-connectors) guidelines
+    - If a **Breaking Change** is being introduced, ensure an Airbyte engineer has created a Breaking Change plan by using the [Breaking Change Release Playbook](https://docs.google.com/document/d/1VYQggHbL_PN0dDDu7rCyzBLGRtX-R3cpwXaY8QxEgzw/edit).
+    - If a **Breaking Change** is being introduced, merge this PR with the 🚨🚨 emoji.
+    - Secrets in the connector's spec are annotated with `airbyte_secret`
+    - The `README.md` is updated
+    - The `bootstrap.md` file is updated. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
```

Comment:
This file doens't exit?  What's bootstrap.md?

Reply:
Hmm that document linked goes no where.

But the files are found in alot of connectors. [Heres an example](https://github.com/airbytehq/airbyte/blob/bnchrch/connector-ops/add-checklist/airbyte-integrations/connectors/source-wrike/bootstrap.md#L8)

But its only referenced in one README.md explicitly
https://github.com/airbytehq/airbyte/blob/bnchrch/connector-ops/add-checklist/airbyte-integrations/connectors/source-fauna/README.md#L3

It looks like the original goal was to have a quick setup guide for experienced airbyte devs.

Since these already exist im thinking we merge this all into one line

```md
- [ ] Ensure all documentation files are up to date. (README.md, bootstrap.md, docs.md, etc...)